Yafc/Workspace/ProductionTable/ProductionLinkSummaryScreen.cs

```diff
@@ -118,6 +205,17 @@ void changeLinkView(ProductionLink newLink) {
         }
     }
 
+    private static SchemeColor GetColor(int c) => (c % 7) switch {
```

Comment:
I would rename to `GetFlowColor()` making it more clear that this is the list of colors meant for the 'flow' field. (or 'embed' in the `BuildFlow()` function)